### PR TITLE
Enable py3/plone5.2 testing

### PR DESCRIPTION
**Pull request by MrTango**

### CODE REVIEW

**Comment by MrTango** [comment]
@cekk @datakurre I'm struggeling with the last remaining error under p37plone52, do you have idea what this could be?

File: plone/app/standardtiles/existingcontent.py [hunk 1]
```diff
@@ -148,7 +148,10 @@ def content_context(self):
                 item = uuidToObject(uuid)
             except Unauthorized:
                 item = None
-                if not self.request.get('PUBLISHED'):
+                if not self.request.get('PUBLISHED') and six.PY2:
```
Comment (petschki) [inline plone/app/standardtiles/existingcontent.py]: @MrTango this is really strange. Unauthorized exception in Py3 doesn't behave the same like in Py2. I could not find out why exactly ... the only thing that looks strange is the fact, that if you step through the code from here, the Py2 exception looks like `Unauthorized: Unauthorized()` and the Py3 exception `AccessControl.unauthorized.Unauthorized: You are not allowed to access 'an-another-page' in this context`

Comment (petschki) [reply plone/app/standardtiles/existingcontent.py]: maybe @datakurre has some insights on this re-raise problem?

Comment (MrTango) [reply plone/app/standardtiles/existingcontent.py]: thx

**Pull request merged**